samcli/commands/pipeline/bootstrap/cli.py

```diff
@@ -265,15 +266,29 @@ def do_cli(
         Stage,
     )
 
+    is_github_parameters_provided = github_org is not None and github_repo is not None
+    is_gitlab_parameters_provided = gitlab_group is not None and gitlab_project is not None
+
+    if permissions_provider == OPEN_ID_CONNECT:
+        if oidc_provider == GITHUB_ACTIONS:
+            if not (is_github_parameters_provided and not is_gitlab_parameters_provided and not bitbucket_repo_uuid):
+                raise click.UsageError("Github Actions can only accept parameters '--github-org' and '--github-repo'")
```

Comment:
could we use the Mutex click option instead?

Reply:
Sure! Actually I found we have a Mutex class after this comment. Will update the code with Mutex click option